src/constants/network.js

```diff
@@ -0,0 +1,14 @@
+const NETWORK = {
+  available: {
+    test: {
+      url: 'https://horizon-testnet.stellar.org',
+      useNetworkFunc: 'useTestNetwork',
```

Comment:
Not sure when this got introduced... by I want to flag it as undesirable.  The 'useXNetwork' functions are helpers for simple scripts... calling to those methods by name  based upon data is needlessly complex and brittle.  Instead... use the `Network.use` method and pass it an instance of `Network`.